gcs/upload.py

```diff
@@ -330,6 +331,9 @@ def process_bidi_write_object_grpc(cls, db, request_iterator, context):
                 # Handles final message with no data to insert.
                 upload.complete = True
                 continue
+            else:
+                print("WARNING unexpected data field %s\n" % data)
```

Comment:
Is this a debugging remnant?  We don't use `print()` anywhere else in the code.  Should we return an error to the client?

Reply:
Right, it makes more sense to return an error. Updated both WriteObject and BidiWriteObject plus tests.